tutorial_src/lea/doc/spiral.cls

```diff
@@ -0,0 +1,128 @@
+\NeedsTeXFormat{LaTeX2e}
+\ProvidesClass{spiral}
+
+\DeclareOption*{\PassOptionsToClass{\CurrentOption}{article}}
+\ProcessOptions\relax
+\LoadClass{article}
+
+\usepackage[a4paper, includeheadfoot, margin=40pt, footskip=\dimexpr\headsep+\ht\strutbox\relax, bmargin = \dimexpr60pt+2\ht\strutbox\relax,]{geometry}
+
+\usepackage[utf8]{inputenc}
+\usepackage{cmbright}
+\usepackage{newtxmath}
+\usepackage{amsfonts, amsmath, amssymb}
+\usepackage[T1]{fontenc}
+
+\usepackage{multirow}
+\usepackage{graphicx}
+\usepackage{tabularx}
+\usepackage{verbatim}
+\usepackage{fancyvrb}
+\usepackage{lastpage}
+\usepackage{bbding}
+\usepackage{tikz}
+\usepackage{hyperref}
+
+% removes section numbers
+\renewcommand{\@seccntformat}[1]{}
+
+
+\def\ifdef#1{\ifcsname#1\endcsname}
+
+% header
+\def\makeheader{%
+  \raisebox{15px}{
+  \hspace{-20px}
+  \begin{minipage}[b][50px]{0.85\textwidth}
+  \hspace{2px}
+  {\LARGE \bf \title} \\
+  \begin{small}
+  \begin{tabularx}{\textwidth}{Xrl}
+    & \ifdef{id} Short name: & \bf \id \fi \\
+    \bf \contest & \ifdef{TL} Time limit: &  \bf \TL \fi \\
+    \desc & \ifdef{ML} Memory limit: & \bf \ML \fi \\
```

Comment:
Tu też trzeba polski/angielski.

Reply:
Tu też problem rozwiążę w osobnym pull requeście, który rozwiąże to dla wszystkich paczek